### ISSUE TITLE
a question on "alpha" param for categorical cross entropy loss

### ISSUE THREAD

**Issue by captainst** [umbertogriffo/focal-loss-keras#20]
From the categorical entropy loss implementation, it seems that the "alpha" array is used to weight the losses among different categories. I.e, the bigger the value is, the loss for that category is weighted more than other categories.

For example, if we have C=3 (from C1 ~ C3), and alpha = [0.2,0.4,0.2], does it mean that loss of C1 is weighted the same as that of C3, while the loss of C2 is weighted twice as much as those of C1 and C3 ?

In this case, **alpha=[0.2,0.4,0.2]** should have the same effect as **alpha=[0.4,0.8,0.4]**
